doc/howto/move_instances.md

```diff
@@ -59,6 +59,12 @@ To do so, ensure the following configuration:
 * Set {config:option}`instance-migration:migration.stateful` to `true` on the instance.
 * Set [`size.state`](devices-disk) of the virtual machine's root disk device to at least the size of the virtual machine's {config:option}`instance-resource-limits:limits.memory` setting.
 
+However, if you create an instance on a shared storage backend like Ceph RBD, you don't have any parameters to set to enable the live migration. This is due to the fact that when the instance is created, it is by default set with {config:option}`instance-migration:migration.stateful` to `true` with the [`size.state`](devices-disk) parameter being unset a this is not strictly required for shared storage.
```

Comment:
```suggestion
If you create an instance on a shared storage backend like Ceph RBD, you don't need to set any parameters to enable the live migration. When an instance is created on a shared storage backend, {config:option}`instance-migration:migration.stateful` is set to `true` by default for the instance. Additionally, the [`size.state`](devices-disk) parameter is unset, because it is not required for shared storage.
```

Reply:
We should probably also update the documentation for the default value of migration.stateful?